public/pages/CreateRollup/utils/helpers.ts

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { FieldItem } from "../models/interfaces";
+
+export const parseTimeunit = (timeunit: string): string => {
+  if (timeunit == "MINUTES" || timeunit == "m" || timeunit == "Minutes") return "minute(s)";
+  else if (timeunit == "HOURS" || timeunit == "h" || timeunit == "Hours") return "hour(s)";
+  else if (timeunit == "SECONDS" || timeunit == "s" || timeunit == "Seconds") return "second(s)";
+  else if (timeunit == "DAYS" || timeunit == "d" || timeunit == "Days") return "day(s)";
+  else if (timeunit == "ms" || timeunit == "Milliseconds") return "millisecond(s)";
+
+  return timeunit;
+};
+
+export const parseTimezone = (timezone: string): string => {
```

Comment:
I don't see this used anywhere, am I just not finding it? Otherwise let's delete things that are not used

Reply:
Removed